<launch>

  <arg name="game_id"   default="0" />

  <node name="cultureid_waypoints_following" pkg="cultureid_waypoints_following" type="cultureid_waypoints_following.py" output="screen">
    <param name="game_id"    value="$(arg game_id)" />
    <rosparam command="load" file="$(find cultureid_waypoints_following)/config/params.yaml" />
  </node>

</launch>
